package-lock.json

```diff
@@ -8417,6 +8417,26 @@
                 "vscode-azurekudu": "^0.1.9",
                 "vscode-nls": "^4.1.1",
                 "websocket": "^1.0.31"
+            },
+            "dependencies": {
+                "vscode-azureextensionui": {
```

Comment:
You see how this is using a different version of the UI package than the one below? We want to avoid that 😬 Whenever you make a breaking change to the UI package you should also update the reference [here](https://github.com/microsoft/vscode-azuretools/blob/master/appservice/package.json#L50) and push out an update to the app service package as well

Reply:
Ok, I'll fix that now. Thanks!